articles/customizing-django-admin/index.md

```diff
@@ -0,0 +1,349 @@
+﻿
+
+
+### Customizing Django Admin
+
+Django is a Web framework for perfectionists with deadlines, it makes sense why it comes with its admin site. Django gives programmers the power of customizing the admin site into suiting the requirements of the project or even the likeness of the client.
+### Prerequisites
+You should at least have the following before we begin:
+
+ - Python installed in your pc
+ - Django installed in your pc
+ - Some basic knowledge in Python 
+ - Visual Studio Code or any other code editor that you will be                                    comfortable with
+
+### Step 1 – Creating our  Django app
+Open your command line and type in the following command to create the project.
+
+   `$ django-admin startproject Custom`
+
+Now let's change our directory to Custom.
+
+`$ cd Custom`
+
```

Comment:
Use the correct  bash commands syntax highlighting syntax .ie.
```markdown
$ cd Custom
```
Please revise your document fixing similar issues